src/components/cms/sections/CalendarSection.tsx

```
@@ -1138,56 +1269,111 @@ export default function CalendarSection({
           
           {/* Step Titles Configuration */}
-          <div>
-            <h5 className="font-medium text-gray-900 mb-3">Step Titles</h5>
+          <div className="border-t pt-4">
+            <h3 className="text-sm font-medium mb-3 flex items-center">
+              <LayoutPanelTop className="h-4 w-4 mr-2 text-muted-foreground" />
+              Step Titles
+            </h3>
             <div className="grid grid-cols-1 md:grid-cols-3 gap-3">
               <div>
                 <label className="block text-sm text-gray-700 mb-1">Location Step</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)